src/Gui/RadiumWindow/SimpleWindow.hpp

```diff
@@ -50,7 +50,7 @@ class RA_GUI_API SimpleWindow : public Ra::Gui::MainWindowInterface
 
   public slots:
     /// Call after loading a new file to let the window resetview for instance.
-    virtual void postLoadFile( const std::string& filename ) override;
+    virtual void prepareDisplay( const std::string& filename ) override;
```

Comment:
can we get rid of this filename here ?

Reply:
Sure.

Reply:
Well, why this parameter is needed ? 
If it's on the App side, then overload in app, but lib do not use it ... simply delete ?

Reply:
Some apps used it to name the root entity. Others use it to generate some informations about scene ... Well, it is now always used and you might forget it if you dont care about scene naming.